aldor/aldor/src/annabs.c

```diff
@@ -198,6 +211,19 @@ abAnnotatedSExpr(AbSyn ab, AbAnnotationBucket bucket)
 		sx = sxNReverse(sx);
 		break;
 		}
+	  case AB_Add:
+	  case AB_With: {
+		if (abcElideInnerExpressions) {
+			sx = sxCons(abInfo(abTag(ab)).sxsym, sxNil);
+		}
+		else {
+			sx  = sxCons(abInfo(abTag(ab)).sxsym, sxNil);
+			for (ai = 0; ai < abArgc(ab); ai++)
+				sx = sxCons(abAnnotatedSExpr(abArgv(ab)[ai], bucket), sx);
```

Comment:
I thought we had an abArg(ab, idx)?

Reply:
Nope, doesn't exist.  Maybe it ought to, but not for this set of changes

Reply:
I see. In one of my rewrite branches before we merged I did add that (with asserts). Maybe another time then.

Reply:
If you have it in a patch or branch I could try to bring it up to date.